tests/rptest/tests/topic_creation_test.py

```diff
@@ -189,7 +189,9 @@ class CreateTopicsTest(RedpandaTest):
         'max.message.bytes':
         lambda: random.randint(1024 * 1024, 10 * 1024 * 1024),
         'redpanda.remote.delete':
-        lambda: "true" if random.randint(0, 1) else "false"
+        lambda: "true" if random.randint(0, 1) else "false",
+        'segment.ms':
+        lambda: random.randint(-1, 10000000),
```

Comment:
This `randint` call includes values outside the permitted range, so it'll probably fail very rarely?